flow-typed/box-ui-elements.js

```diff
@@ -18,6 +18,7 @@ import type { ContentSidebarProps } from '../src/elements/content-sidebar';
 import type { ContentOpenWithProps } from '../src/elements/content-open-with';
 import type { ContentPreviewProps } from '../src/elements/content-preview';
 import type { FeatureConfig } from '../src/elements/common/feature-checking';
+import type { TaskNew } from '../src/elements/content-sidebar/activity-feed/task-new/flowTypes';
```

Comment:
Is this path a good place to put this if we also update the `api/tasks/*` and `api/feed.js` files to reference these types?

Reply:
Done, moved to the `api` folder.